lenient/src/demo.c

```diff
@@ -27,6 +27,7 @@ demo_construct_to_heap(int _foo)
     struct demo* instance_public;
     struct demo_private* instance_private;
 
+    // TODO initialize remaining local variables to NULL
```

Comment:
Please implement in a dedicated commit, even though this does not belong to issue being implemented.

Reply:
Implemented in: 04bb370a9f52606085ec06f972d4af5959f23361